pom.xml

```diff
@@ -12,22 +12,17 @@
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
 
-    <app.name>service-a</app.name>
-    <testServerHttpPort>9080</testServerHttpPort>
-    <testServerHttpsPort>9443</testServerHttpsPort>
+    <app.name>${project.artifactid}</app.name>
+    <testServerHttpPort>8080</testServerHttpPort>
+    <testServerHttpsPort>8443</testServerHttpsPort>
     <warContext>${app.name}</warContext>
```

Comment:
Are these Liberty specific properties?

Reply:
Yes. I'll move them.

Reply:
Thanks.

Are `package.file`, `packaging.type` and `warContext` also Liberty specific?